src/Mod/TechDraw/App/DrawViewPart.cpp

```diff
@@ -1498,6 +1498,40 @@ void DrawViewPart::handleChangedPropertyType(Base::XMLReader &reader, const char
     }
 }
 
+// true if owner->element is a cosmetic vertex
+bool DrawViewPart::isCosmeticVertex(App::DocumentObject* owner, std::string element)
+{
+    auto ownerView = static_cast<TechDraw::DrawViewPart*>(owner);
+    auto vertexIndex = DrawUtil::getIndexFromName(element);
+    auto vertex = ownerView->getProjVertexByIndex(vertexIndex);
+    if (vertex) {
+        return vertex->getCosmetic();
+    }
+    return false;
+}
+
+// true if owner->element is a cosmetic edge
+bool DrawViewPart::isCosmeticEdge(App::DocumentObject* owner, std::string element)
+{
+    auto ownerView = static_cast<TechDraw::DrawViewPart*>(owner);
+    auto edge = ownerView->getEdge(element);
```

Comment:
Cool, now code location is good. Since this is now a member function of DrawViewPart, we no longer need to pass a DrawViewPart to the function (as it is already located in one).

```suggestion
bool DrawViewPart::isCosmeticEdge(std::string element)
{
    auto edge = getEdge(element);
```